examples/tikhonov_regularization/example_tik_multi_lam.cpp

```diff
@@ -97,6 +112,11 @@ void run(size_t m, size_t n, size_t k)
         sqrt(normr_inside0 * normr_inside0 + normr_inside1 * normr_inside1 -
              lambda * lambda * normx_inside * normx_inside);
 
+    // at this point we have ||b-Ax||₂ and ||x||₂ and can decide for a new
```

Comment:
change to "at this point we have ||b-Ax||₂ ( in normr_inside ) and ||x||₂ ( in normx_inside )"